### PR TITLE
Filter topics in topic selection menu based on team membership

### PR DESCRIPTION
**Pull request by rowanseymour**
Note this doesn't filter tickets in the non-topic specific folders.

### CODE REVIEW

File: temba/tickets/models.py [hunk 1]
```diff
@@ -81,6 +81,18 @@ def create(cls, org, user, name: str):
...
 
+    @classmethod
+    def get_accessible(cls, org, user):
+        """
+        Gets the topics accessible to the given user in the given org.
+        """
+
+        membership = org.get_membership(user)
+        if membership.team and not membership.team.all_topics:
+            return membership.team.topics.all()
```
Comment (norkans7) [inline temba/tickets/models.py:92]: Here we should as well filter by `is_active=True`?

Comment (rowanseymour) [reply temba/tickets/models.py:92]: when topics are released they're removed from teams

Comment (norkans7) [reply temba/tickets/models.py:92]: Ok

Comment (norkans7) [reply temba/tickets/models.py:92]: could that be safer to have the filter?
what could happen if a released topic is included?

Comment (rowanseymour) [reply temba/tickets/models.py:92]: I can add it but with these sorts of things I weigh up the risk of code not working as I expect vs are we making harder to spot that the code isn't working. In this case a user seeing a topic they just deleted isn't the end of the world. This risk is obviously different if we're talking about filtering by org or something like that.

**Pull request merged**